cmd/agent/main.go

```diff
@@ -169,6 +170,15 @@ func main() {
 		if err != nil {
 			log.Fatalf("[ERROR] [main,nomad] [message: Unable to retrieve local IP associated to the agent] [error: %s]", err)
 		}
+
+		nomadConfig.NomadAddr = goos.Getenv(agent.NomadAddrEnvVarName)
```

Comment:
Where are these variables coming from? Set by the user when deploying the agent or available in the environment as part of a Nomad installation?

Reply:
These two environment variables are set as part of a Nomad installation.